frontend/src/views/WheresMyGene/common/store/reducer.ts

```diff
@@ -171,6 +171,10 @@ function selectOrganism(
     selectedTissues: EMPTY_ARRAY,
     selectedFilters: EMPTY_FILTERS,
     cellInfoCellType: null,
+    geneInfoGene: null,
```

Comment:
what do you think of this?

```ts
function selectOrganism(
  state: State,
  action: PayloadAction<string | null>
): State {
  if (state.selectedOrganismId === action.payload) {
    return state;
  }

  const { snapshotId } = state;

  return {
    ...INITIAL_STATE,
    snapshotId,
    selectedOrganismId: action.payload,
  };
}
```

Reply:
state to carry over wasn't complete, but I implemented